quill-cassandra/src/main/scala/io/getquill/context/cassandra/CqlIdiom.scala

```diff
@@ -23,7 +23,7 @@ object CqlIdiom {
       case Infix(parts, params) =>
         StringContext(parts: _*).s(params.map(_.show): _*)
       case a @ (
-        _: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation |
+        _: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation | _: OptionProperty |
```

Comment:
no, cassandra doesn't have the concept of NULL and CQL doesn't have a IS
NULL/IS NOT NULL operator

On 31 Jul 2016 15:59, "Gustavo Amigo" <notifications@github.com> wrote:

> In
> quill-cassandra/src/main/scala/io/getquill/context/cassandra/CqlIdiom.scala
> <https://github.com/getquill/quill/pull/485#discussion_r72905637>:
>
> > @@ -23,7 +23,7 @@ object CqlIdiom {
> >        case Infix(parts, params) =>
> >          StringContext(parts: _*).s(params.map(_.show): _*)
> >        case a @ (
> > -        _: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation |
> > +        _: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation | _: OptionProperty |
>
> Doesn't CQL support IS NOT NULL as well?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/getquill/quill/pull/485/files/4390cdc783020489d8b50e498fed20ba31a2c388#r72905637>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADxcVaqny4oVXfR6QZXXLRHZ3Ec2mp6mks5qbLhlgaJpZM4JX21d>
> .
>